### PR TITLE
Sending Assets for all created Resources - sls app

### PR DESCRIPTION
**Pull request by Johnbush**
### Requirements

* Sending Assets details for all resources which are based out of serverless Application Templates

### Description of the Change

- Assets details are fetched using the cloud formation templates and updated to assets tables successfully.

### CODE REVIEW

File: builds/jazz-build-module/sls-app/whitelist.yml [hunk 1]
```diff
@@ -462,3 +462,14 @@ plugins:
   - serverless-mocha-plugin
   - serverless-kms-secrets
     
+arnTemplate:
```
Comment (raghits) [inline builds/jazz-build-module/sls-app/whitelist.yml]: Need to check what other resource types we need to add to assets and add here.

File: builds/jenkins-build-pack-sls-app/Jenkinsfile [hunk 2]
```diff
@@ -406,6 +400,33 @@ def deleteAndRedeployService(environment_logical_id, envBucketKey, accountDetail
...
+  } catch (ex){
+    echo "Exception occured on getting Arn templates for Resource Type : ${resourceType}"
+    error "Exception occured on getting Arn templates for Resource Type : ${resourceType}"
+  }
+  return arn
+}
+
+def getStackResources(env){
+  def cfStackName = "${instance_prefix}-${config['domain']}-${config['service']}-${env}"
+  def stackResources = sh(script: "aws cloudformation describe-stack-resources --stack-name ${cfStackName} --region ${config['region']}", returnStdout: true)
```
Comment (SiniWilson) [inline builds/jenkins-build-pack-sls-app/Jenkinsfile]: use the profile

File: builds/jazz-build-module/sls-app/whitelist.yml (hunk 1 above)
Comment (SiniWilson) [reply builds/jazz-build-module/sls-app/whitelist.yml]: Please check the dynamodb stream, cloudwatch event, kinesis stream, and sqs also

File: builds/jenkins-build-pack-sls-app/Jenkinsfile (hunk 2 above)
Comment (Johnbush) [reply builds/jenkins-build-pack-sls-app/Jenkinsfile]: updated.

**Pull request merged**